dags/merino_jobs.py

```diff
@@ -35,6 +36,10 @@ def merino_job(
         location="us-west1",
         cmds=["python", "-m", "merino.jobs.cli"],
         arguments=arguments,
+        container_res = k8s.V1ResourceRequirements(
+            requests={"memory": "6000Mi"},
+            limits={"memory": "8000Mi"},
```

Comment:
We can set the `requests` and omit the `limits` so it can use up to what the cluster allows. Also, request from 6GiB seems a little too high, we can set it to `512Mi` to begin with.